src/App/Providers/AppServiceProvider.php

```diff
@@ -23,5 +24,16 @@ public function boot(): void
         Factory::guessFactoryNamesUsing(function (string $modelName) {
             return '\Database\Factories\\'.class_basename($modelName).'Factory';
         });
+
+        Relation::enforceMorphMap([
+            'item' => 'Domain\Items\Models\Item',
```

Comment:
```suggestion
            'item' => Item::class,
```
If we declare the value like so, instead of a string, we can easily find and replace in the future, in addition to the IDE letting us click into it or when renaming the file, the IDE will know what to do here.

Reply:
Ah just realised CW said the same thing below

Reply:
Yeah spot on! I hadn't realised you could do that (since the docs use the string values), so I'll make the change :)